mmv1/products/bigqueryconnection/Connection.yaml

```diff
@@ -125,6 +125,25 @@ examples:
     primary_resource_id: "connection"
     vars:
       connection_id: "my-connection"
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'bigquery_connection_kms'
+    pull_external: true
+    primary_resource_id:
+      'connection'
+    # Random provider
+    skip_vcr: true
```

Comment:
Can we avoid using the random provider here so these test can run in the VCR?

If you use a variable with `-` or `_` in the name we attach a random suffix to it. ie: `my-password`

Reply:
@ScottSuarez Removed it